FashionConnect_ui/src/components/EditProfile/EditProfile.jsx

```diff
@@ -1,39 +1,45 @@
 import "./EditProfile.css";
 import React, { useState,useContext } from "react";
 import { Link, useNavigate } from "react-router-dom";
-
+import { UserContext } from '../../UserContext.jsx';
+// import { response } from "express";
```

Comment:
Let's also make sure to remove any commented out code before you put up PRs.

Reply:
My process before I usually put up PRs is I go through every file I've changes and I compare what the previous file was and what changes I've made. Then I remove any console.log statements or commented out code. I also make any changes to indents etc. Once I've done all this clean up, I proceed to make a PR.

Reply:
Thanks for this, i will start doing this as from now.